infrastructure/lambda/private-api.yaml

```diff
@@ -126,6 +126,49 @@ paths:
           Fn::Sub: arn:${AWS::Partition}:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/arn:${AWS::Partition}:lambda:${AWS::Region}:${AWS::AccountId}:function:${CommonStackName}-AuthorizationFunction/invocations
         passthroughBehavior: "when_no_match"
 
+  /person-info:
+    get:
+      parameters:
+        - in: header
+          name: "session_id"
+          schema:
+            type: string
+            format: uuid
+          required: true
+        - $ref: '#/components/parameters/AuditHeader'
+      requestBody:
```

Comment:
@ElliotMurphyGDS do we need the requestBody section if this is a GET call now?

Reply:
Good catch